test/webapi/ows/stac/stac-item.json

```diff
@@ -0,0 +1,1150 @@
+{
+  "stac_version": "1.0.0",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/datacube/v2.1.0/schema.json"
+  ],
+  "type": "Feature",
+  "id": "demo-1w",
+  "bbox": [
+    0.0,
+    50.0,
+    5.0,
+    52.5
+  ],
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          0.0,
+          50.0
+        ],
+        [
+          0.0,
+          52.5
+        ],
+        [
+          5.0,
+          52.5
+        ],
+        [
+          5.0,
+          50.0
+        ],
+        [
+          0.0,
+          50.0
+        ]
+      ]
+    ]
+  },
+  "properties": {
+    "datacube:dimensions": {
+      "lon": {
+        "type": "spatial",
+        "axis": "x",
+        "description": "longitude",
+        "unit": "degrees_east",
+        "extent": [
+          0,
+          5
+        ],
+        "step": 0.0025,
+        "reference_system": "EPSG:4326"
+      },
+      "lat": {
+        "type": "spatial",
+        "axis": "y",
+        "description": "latitude",
+        "unit": "degrees_north",
+        "extent": [
+          50,
+          52.5
+        ],
+        "step": 0.0025,
+        "reference_system": "EPSG:4326"
+      },
+      "time": {
+        "type": "temporal",
+        "values": [
+          "2017-01-22T00:00:00Z",
+          "2017-01-29T00:00:00Z",
+          "2017-02-05T00:00:00Z"
+        ]
+      }
+    },
+    "datacube:variables": {
```

Comment:
Datacube schema specifies `cube:`, not `datacube:` -- typo?

Reply:
See above.